apps/frontend/src/components/atoms/AskForCredits.tsx

```diff
@@ -1,16 +1,15 @@
-import { EXTERNAL_ROUTES, Button } from '@auto-drive/ui';
+import { Button, ROUTES } from '@auto-drive/ui';
+import { InternalLink } from './InternalLink';
+import { useNetwork } from '../../contexts/network';
 
-export const AskForCreditsButton = () => {
+export const BuyMoreCreditsButton = () => {
```

Comment:
It's confusing to have this file named ask for credits when it's now about buying credits.
This would also need to change in the nav bar.